src/UKHO.ERPFacade.Common/IO/XmlHelper.cs

```diff
@@ -12,5 +13,30 @@ public XmlDocument CreateXmlDocument(string xmlPath)
             xmlDocument.Load(xmlPath);
```

Comment:
We already have commonhelper.cs file for xml related helper. Please keep either of the file and move all methods in single helper class.

Reply:
@Vishal14583 This file XmlHelper.cs and method CreateXmlDocument is already used by the existing code of ERP facade. Thats why we did not remove it and added another xml related method here only.